### ISSUE TITLE
Convert Raw JSON To Python Objects

### ISSUE THREAD

**Issue by hydrosquall** [hydrosquall/tiingo-python#31]
Rather than forcing users to work directly with JSON objects, I would like to convert these JSON Dicts into Python Objects. These Resources might include

- `NewsItem`
- `PriceEvent`
- `StockTicker`

Each object should have a method that enables the "serialized" version of that object to be the JSON dict that produced it. This could be handled with Peewee and/or Marshmallow.

A separate issue/PR might be filed to write documentation around how to use each of these resources.

**Comment by BharatKalluri** [comment]
Hello, Can I try to fix this?

**Comment by hydrosquall** [comment]
Hi @BharatKalluri! Thanks for checking out this project, that would be awesome!

Presently `fmt` controls whether the endpoint returns `csv` or `json`. for the `get_ticker_price` method, but not for any of the others (they just always return `json`).

I'm not sure what would be the best way to toggle whether this client returns the raw data formats, versus returning Python Objects. Maybe a format called "object"? Or a new boolean param called "raw" which is False by default?

Please feel free to rename the items I came up with if you have alternate suggestions. I imagine they might live under a folder called `models` in the main `tiingo.py` folder.

Further docs about the properties available for each object are on the official API website: https://api.tiingo.com/docs/iex/realtime

**Comment by Fux002** [comment]
Hi,

This function does t work:
tickers = client.get_stock_tickers()

any clue why this is:

Traceback (most recent call last):
get_available_tickers
    return client.get_stock_tickers()
AttributeError: 'TiingoClient' object has no attribute 'get_stock_tickers'
[Finished in 0.6s]

**Comment by Fux002** [comment]
How do i pull historical data please?

Can you give me an example

**Comment by hydrosquall** [comment]
Hi @Fux002! Thanks for your questions- in the future, please feel free to open a new issue using the green `New Issue`button in the top right of this page.

As to your questions:

There was a typo in the docs, I misnamed the method. It should actually be `list_stock_tickers` instead of `get_stock_tickers`.

```
tickers = client.list_stock_tickers()
```

I'll update the example code in the next release.

Regarding historical data, you could get the tickers from Google from the first quarter of last year using the existing snippet in the docs (feel free to adjust the date ranges). Note you can choose between resample frequency values of `daily`, `weekly`, `monthly`, or `annually`.

```python
prices = client.get_ticker_price("GOOGL", frequency="weekly", 
                                                      startDate='2017-01-01',
                                                      endDate='2017-04-01')
```

Hope this helps!

**Comment by hydrosquall** [comment]
Fixed by #33 . Thanks @BharatKalluri !
